.circleci/config.yml

```diff
@@ -1,13 +1,19 @@
 version: 2.1
+
+defaultImage: &defaultImage
+  image: "quay.io/rhacs-eng/apollo-ci:snapshot-jenkins-plugin-0.3.26-19-g230e4fc30a" # TODO(do not merge): After upstream PR is merged, cut a tag and update this
```

Comment:
What PR needs to be merged?

Reply:
https://github.com/stackrox/rox-ci-image/pull/107